SignalShareExtension/ShareViewController.swift

```diff
@@ -90,18 +90,18 @@ public class ShareViewController: UIViewController, ShareViewDelegate, SAEFailed
         // We shouldn't set up our environment until after we've consulted isReadyForAppExtensions.
         AppSetup.setupEnvironment({
```

Comment:
nit:  renaming method to `setupEnvironmentWithCallMessageHandlerBlock:` would make this more clear.

Reply:
Good idea, done.